### DISCUSSION TITLE
Need to convert for debian (https://github.com/OE4T/meta-tegra-community.git) and deploy on Debian Image.

### DISCUSSION THREAD

**Discussion by PugalendhiMurugan** [OE4T/meta-tegra-community#135] [Q&A]
Hi All,
Actually I need to add meta-tegra community layer for debian Image for testing purpose. Kindly let me is there way to convert meta-tegra community compatible for debian.
Thanks,
Pugalendhi M

**Comment by madisongh** [comment]
If you're running a Debian-based distro, you're probably better off using the NVIDIA package feeds directly. You can have a Yocto build that generates Debian-style packages - see [here](https://docs.yoctoproject.org/ref-manual/variables.html#term-PACKAGE_CLASSES) in the Yocto project docs - but they're not necessarily going to be compatible with real Debian/Ubuntu/whatever.

**Comment by ichergui** [comment]
Hi @PugalendhiMurugan 
Could you please elaborate on your request ? 
Sorry. I didn't understand
